addons/mail/static/tests/new/composer/composer_tests.js

```diff
@@ -44,4 +46,34 @@ QUnit.module("mail", (hooks) => {
             assert.containsOnce(target, ".o-mail-emoji-picker");
         }
     );
+
+    QUnit.test("can post a message", async (assert) => {
```

Comment:
test title looks too generic if the goal is to test non-send of duplicate message